### DISCUSSION TITLE
How about node detail?

### DISCUSSION THREAD

**Discussion by xu1919** [daeuniverse/dae#556] [Q&A] [answered]
我想从clash换到dae，但node的具体写法不懂，有没有人知道ss等的具体写法？

**Comment by Mapleshade20** [comment, marked as answer]
ss://PASSWORD@node.global.xyz:10002/?plugin=simple-obfs%3Bobfs%3Dhttp%3Bobfs-host%3D0ee7c3782f.m.ctrip.com#%E6%97%A5%E6%9C%AC%E5%AE%9E%E9%AA%8C%E6%80%A7%20IEPL%20%E4%B8%AD%E7%BB%A7%201
(没有obfs就不要/?后的)

**Comment by oluceps** [comment]
https://github.com/shadowsocks/shadowsocks-org/wiki/SIP002-URI-Scheme
